pkg/ioutils/concat.go

```diff
@@ -52,20 +52,20 @@ func (self *concatReadSeekCloser) Read(p []byte) (n int, err error) {
 			return 0, err
 		}
 
-		nA, err := io.ReadFull(self.a, p[:min(len(p), int(self.aSize - self.off))])
+		nA, err := io.ReadFull(self.a, p[:min(len(p), int(self.aSize-self.off))])
```

Comment:
This is personal preference, but in general I find spaces around infix operators more readable.

Reply:
@dfunckt go has a language imposed style so we'll go with it. https://blog.golang.org/go-fmt-your-code